packages/okta-vue/src/components/ImplicitCallback.vue

```diff
@@ -0,0 +1,10 @@
+<template></template>
+
+<script>
+export default {
+  name: 'ImplicitCallback',
+  async beforeMount () {
+    this.$router.push({ path: await this.$auth.handleAuthentication() })
```

Comment:
The method `$auth.handleAuthentication()` is returning a URL string, which is not obvious from the name of this method.  I think it would be better if this method did not return any value, and instead handle the URL need separately.  Something like:

```
await this.$auth.handleAuthentication()
this.$auth.postLoginRedirect()
```